tox.ini

```diff
@@ -11,11 +11,12 @@ setenv =
     PYTHONHASHSEED = 42
 commands =
     python setup.py build_ext --inplace
-    py27,py34,py35: nosetests -v zarr
-    py36: nosetests -v --with-coverage --cover-erase --cover-min-percentage=100 --cover-package=zarr --with-doctest --doctest-options=+NORMALIZE_WHITESPACE zarr
+    py27: nosetests -v --with-coverage --cover-erase --cover-package=zarr zarr
+    py34,py35,py36: nosetests -v --with-coverage --cover-erase --cover-package=zarr --with-doctest --doctest-options=+NORMALIZE_WHITESPACE zarr
```

Comment:
Yeah, I don't understand. It's something new in PY36, works fine in both PY34 and PY35. I know dict implementation has changed in PY36, guessing that has something to do with it.